DotaStats/AppCoordinator.swift

```diff
@@ -24,15 +25,36 @@ final class AppCoordinator {
         tabBarController.tabImageNames = tabImageNames
 
         tabBarController.configureTabs()
+        setupNavigationBarAppereance()
+    }
+
+    private func createNavigationController(for viewController: UIViewController,
+                                            with title: String) -> UINavigationController {
```

Comment:
Кажется, что такая сигнатура функции делает ее назначение чуть более понятнее
```suggestion
    private func createNavigationControllerWithRootVIewController(_ viewController: UIViewController,
                                            title: String) -> UINavigationController {
```